### DISCUSSION TITLE
[Example][Server] MediaRecorder is not saving audio (mp3) from the web

### DISCUSSION THREAD

**Discussion by niklas-pernter** [aiortc/aiortc#1063] [Q&A]
# Summary 

Hi. I tinkering around with the examples but found a little bug when having "--record-to" flag set to a file e.g. "audio.mp3". Somehow when using the Frontend (only audio no data-channel and no STUN) the file doesn't get saved. 

Maybe I'm doing something wrong but I've tried all different combinations and also tried out inside the code - unfortunately without any success

# How to reproduce

1. Comment out the line 143 with `pc.addTrack(player.audio)` 
2. start python server `python server.py -v --record-to test.mp3`
3. Disable "data-channel" and "video" (keep audio enabled)
4. Click Start
5. Talk into the Microphone
6. Click Stop

# Expected results
A file called "test.mp3" should appear in the root directory with the same words you talked into the mic

# Actual results
No file gets stored and no error comes up

**Comment by el07694** [comment]
write permissions in folder?

**Comment by el07694** [comment]
TRY LAUNCH CMD AS ADMINISTRATOR.

**Comment by jlaine** [comment]
I'm sorry but I don't understand what you are doing, specifically: why are you changing the code for this use case?

On the *unmodified* code I did:

- run `python examples/server/server.py  --record-to test.mp3`
- connect to the web interface and press "start"
- after some time press "stop"
- I do have a file called `test.mp3`
